src/UI/Pages/HomePage/Homepage.tsx

```diff
@@ -45,8 +44,8 @@ const Homepage: React.FC<Props> = ({ viewportHeight }) => {
                     </div>
                     <div className="imgContainer">
                         <CircleFrame
-                            width={window.innerWidth > 768 ? '474px' : '256px'}
-                            height={window.innerWidth > 768 ? '474px' : '256px'}
+                            width={viewportWidth > 768 ? '474px' : '256px'}
+                            height={viewportWidth > 768 ? '474px' : '256px'}
```

Comment:
Here you should use react-responsive (please remind me to speak with you about the motivation of this).